core/src/components/button/button.ionic.vars.scss

```diff
@@ -1,125 +0,0 @@
-@import "../../themes/native/native.globals.ios";
```

Comment:
Why was this file removed?

Reply:
Besides, is importing stuff from a different Design System and Theme (ios).